types/easeljs/index.d.ts

```diff
@@ -924,6 +941,52 @@ declare namespace createjs {
 
     }
 
+    export class StageGL extends Stage {
+        constructor(canvas: HTMLCanvasElement | string | Object);
```

Comment:
Would be nice to define an interface for the options Object with the applicable properties. It also takes a second optional arg of options.

Reply:
forget about it, fixed)